package.json

```diff
@@ -26,8 +26,8 @@
     "serve-favicon": "~2.3.0"
   },
   "devDependencies": {
-    "chai": "^3.5.0",
```

Comment:
This is no good. Revert this change entirely. Chai and mocha were already on the proper versions.